webpack.dev.config.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path');
+const { createConfig } = require('@edx/frontend-build');
+/** Uncomment the lines below if you wish to use an env.config.js in development */
+// const envConfig = require('./env.config');
+
+const config = createConfig('webpack-dev');
+
+config.resolve.modules = [
+  path.resolve(__dirname, './src'),
+  'node_modules',
+];
+
+// config.devServer.port = envConfig.PORT || process.env.PORT || 8080;
```

Comment:
what's up with this commented line?

Reply:
[see comments above](https://github.com/openedx/frontend-app-learner-record/pull/275/files#r1480641752) about why this is currently commented out